src/components/MainTitle.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <h1 class="header-section">
+      {{ text }} <span>{{ subText }}</span>
+    </h1>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "MainTitle",
+  props: {
+    text: {
+      type: String,
+      required: true,
+    },
+    subText: {
+      type: String,
+      required: false,
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+@media (min-width: 0px) {
```

Comment:
actually all of the screens are wider than 0px so you can probably skip `@media` 😉 

I mean
```css
@media (min-width: 0px) {
  /* SOME CODE */
}
```
is equivalent to
```css
/* SOME CODE */
```